CppcheckTargets.cmake

```diff
@@ -27,19 +27,29 @@ endif()
 set(__add_cppcheck YES)
 
 if(NOT CPPCHECK_FOUND)
-	find_package(cppcheck QUIET)
+	find_package(cppcheck)
 endif()
 
-if(CPPCHECK_FOUND)
-	if(NOT TARGET all_cppcheck)
-		add_custom_target(all_cppcheck)
-		set_target_properties(all_cppcheck PROPERTIES EXCLUDE_FROM_ALL TRUE)
-	endif()
+if(NOT CPPCHECK_FOUND)
+  add_custom_target(cppcheck
+    COMMENT "cppcheck executable not found")
```

Comment:
Does this actually work the way I'd expect by reading it? Does it result in a build success or a build failure?